### ISSUE TITLE
ToDo

### ISSUE THREAD

**Issue by konegen** [Hahn-Schickard/Automatic-Structured-Pruning#9]
- [x] EmbeddedPC Pfad hinzufügen, ohne Quantisierung.
- [x] FPGA und EmbeddedPC model memory entfernen.
- [x] Dataloader Python script in README verlinken
- [x] Startfenster Beschriftung Buttons. (Beschriftung in den Buttons?)
- [x] Autokeras Constraints herausnehmen.
- [x] Autkeras Pfad Dateiaufrufe abändern.
- [x] download Funktion Name ändern
- [x] Helper Datei aufspalten --> Dataloader, GUI Helper
- [x] Dataloaderbeispiel für MNIST
- [x] Dataloader für Autokeras einbauen.
- [x] Bilder für AutoKeras Tasks
- [x] Fehlende Tooltips einfügen
- [x] Autokeras Pfad anpassen
- [x] Nicht benötigte Dateien/Fenster aussortieren
- [x] Shape of data for regression
- [x] Pruning funktioniert für AutoKeras Regressionsmodelle nicht
- [x] Compiler beim Trainieren (nach dem Pruning) von Regressionsmodellen
- [x] Bild für EmbeddedPC Button (Für alle Targets, freie Bilder)
- [x] Deactivate tasks which are not working
- [x] Ventilbeispiel kann nicht konvertiert werden
- [x] Developer documentation. --> FAQ bspw. Change GUI layout and events, add device, add optimization, add/modify dataloader
- [x] Test auf Ubuntu
- [x] Dokumentation für Autokeras Pfad, FPGA, EmbeddedPC
- [x] Dokumentation was mit Dateien für MCU, FPGA, EmbeddedPC gemacht werden kann.
- [x] Für MCU, EmbeddedPC Funktion die aufgerufen werden, um Modell auszuführen, über Codeboxen in Readme beschreiben.
- [x] Tensorflow pruning durchführen, wenn eigenes Pruning nicht funktioniert.
- [x] Datei-, Variablen-, Funktionsnamen ändern
- [ ] AutoKeras Dataloader Probleme mit .png Bildern? "Cleanup called..." wird durchgehend ausgegeben???
- [ ] Genauigkeit der Modelle in GUI anzeigen (AutoKeras + nach Optimierung)
- [ ] Accuracy pruning train und test Daten? (für Dataloader und Fileloader unterschiedliche Funktionen?)
- [ ] Accuracy pruning mit Regressionsmodell (Compiler metrik verwenden)?
- [ ] AutoKeras Modellaufbau ist nur sehr limitiert in der Auswahl der Hyperparameter (Anaconda3\envs\AUTOflow\Lib\site-packages\autokeras\blocks\basic.py)
- [ ] pruning_for_accuracy --> Problem bei max_loss_acc: Evaluierung auf den Trainingsdaten, später allerdings mit den Validierungsdaten verglichen --> train_test_split vornehmen --> Split erfolgt allerdings aus den Trainingsdaten?
- [ ] Bei Accuracy Pruning das bereits geprunte Modell für den nächsten Pruningschritt verwenden?
